concourse/pipelines/artifact-releaser-autopush.yaml

```diff
@@ -157,6 +157,21 @@ jobs:
           publish_version: ((.:version))
           source_version: "v20211027"
           release_notes: "Disregard this release. Alma Linux 8 test."
+      - task: publish-prod-debian-11
```

Comment:
hm, i don't think it's really publish prod, is it? my understanding of the PR is it is to publish for testing purposes

Reply:
It's to test publish in ARLE's production environment, would `publish-debian-11-prod-env` work better?